src/harness/security_testcase.py

```diff
@@ -250,3 +250,38 @@ def _InfoCb(conn, where, ok):
       self.assertEquals(client_ssl.get_peer_finished(), None)
     finally:
       client_ssl.close()
+
+  def create_short_lived_certificate(self, client_type, cert_name, cert_expiry_time):
```

Comment:
Please use the same naming style as other functions in this file.

Reply:
Changed   to camelCase.